### PR TITLE
Ui changes

### PR DESCRIPTION
**Pull request by Felixim0**
Updated the following:

FSSS IN 
	Unique employee id
	Country
	Preferred name
	ID Badge Number
FSS OUT
	id badge status

HR IN
	employee id
	Preferred name
	-county
	Country
	Address
	Role id
	Job title short
	Line manager
	Badge number
HR OUT
	Operational start & end dates
	Diversity info - remove
	Operational start date - not in DB - remove
	Operational end date - remove

LOGISTICS IN 	
        Job title short 	
        Line Manager first & last name
LOGISTICS OUT 	
        Operational start & end dates 	
        Diversity info

RMT IN
	JobRole type 
	JobRole

### CODE REVIEW

File: app/views/fsss_view.py [hunk 1]
```diff
@@ -87,7 +89,7 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     data_other = {
         'Job Role Type': current_job_role['jobRoleType'],
-        'Badge Number': get_emp_info('idBadgeNo'),
+        #'Badge Number': get_emp_info('idBadgeNo'),
```
Comment (SammyPoot) [inline app/views/fsss_view.py]: This line can be fully removed

File: app/views/fsss_view.py [hunk 2]
```diff
@@ -51,8 +52,9 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     data_job_role = {
         'Job Role ID': current_job_role['uniqueRoleId'],
-        # badge number
+        'Badge Number': get_emp_info('idBadgeNo'),
```
Comment (SammyPoot) [inline app/views/fsss_view.py]: If the badge number was already present, it seems to me it should stay inside data_other.

File: app/views/hr_view.py [hunk 3]
```diff
@@ -37,6 +37,7 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
         'Preferred Name': preferred_name,
         'Address': get_emp_info('address'),
         'Postcode': get_emp_info('postcode'),
+        'Country': get_emp_info('country'),
```
Comment (SammyPoot) [inline app/views/hr_view.py:40]: FWMT-2682 wants 'County' to also be placed in the HR view

File: app/views/logistics_view.py [hunk 4]
```diff
@@ -33,8 +29,9 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     emp_job_role = {'Job Role ID': current_job_role['uniqueRoleId'],
                     'Job Role': current_job_role['jobRole'],
-                    'Badge Number': get_emp_info('idBadgeNo'),
+                    'Job Role Type': current_job_role['jobRoleType'],
```
Comment (SammyPoot) [inline app/views/logistics_view.py]: I'm not sure this has anything to do with the 'job title short' mentioned in FWMT-2685

File: app/views/rmt_view.py [hunk 5]
```diff
@@ -32,10 +32,11 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     emp_job_role = {
             'Job Role ID': current_job_role['uniqueRoleId'],
-            'Badge Number': get_emp_info('idBadgeNo'),
-            'Postcode': get_emp_info('postcode'),
+            'Job Role': current_job_role['jobRole'],
             'Job Role Short': current_job_role['jobRoleShort'],
             'Job Role Type': current_job_role['jobRoleType'],
+            'Badge Number': get_emp_info('idBadgeNo'),
+            'Postcode': get_emp_info('postcode'),
```
Comment (SammyPoot) [inline app/views/rmt_view.py]: I'm not sure why these were added, they're not mentioned in any ticket I can find.

**Review by SammyPoot — CHANGES_REQUESTED** [review]

File: app/views/fsss_view.py [hunk 6]
```diff
@@ -25,8 +25,9 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
     preferred_name = get_emp_info('preferredName', on_false='None')
 
     data_detail = {
+        'Unique Employee ID': get_emp_info('uniqueEmployeeId'),
         'Name': employee_name,
-        'Preferred Name': preferred_name,
+        'Preferred Name': preferred_name,   # Asked to be added in FWMT-2681 but already present
```
[the remark this answers is not in the document]
Comment (Felixim0) [reply app/views/fsss_view.py]: removed

File: app/views/fsss_view.py (hunk 1 above)
Comment (Felixim0) [reply app/views/fsss_view.py]: Removed

File: app/views/fsss_view.py (hunk 2 above)
Comment (Felixim0) [reply app/views/fsss_view.py]: Moved back to where it should be

File: app/views/logistics_view.py (hunk 4 above)
Comment (Felixim0) [reply app/views/logistics_view.py]: Have removed this line

File: app/views/rmt_view.py (hunk 5 above)
Comment (Felixim0) [reply app/views/rmt_view.py]: I`m not too sure either :| 
Have removed

**Review by SammyPoot — APPROVED** [review]

**Pull request merged**